Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM centos:7
+
+MAINTAINER https://github.com/pettazz/pygooglevoice
+
+ARG user_home=/home/pygooglevoice
+ARG user_name=pygooglevoice
+ARG user_uid=808
```

Comment:
Is there a reason to pick this id or just kind of random?

Reply:
No reason. I can remove the UID from the useradd line below.  CentOS should then pick one by itself.